tools/src/main/scala/scala/scalanative/codegen/Generate.scala

```diff
@@ -115,6 +116,31 @@ object Generate {
       )
     }
 
+    def genClassInitializers(): Unit = {
+      val insts          = mutable.UnrolledBuffer.empty[Inst]
+      implicit val fresh = Fresh()
+
+      insts += Inst.Label(fresh(), Seq())
+
+      defns.foreach {
+        case Defn.Define(_, name: Global.Member, _, _) if name.sig.isClinit =>
+          insts += Inst.Let(Op.Call(Type.Function(Seq(), Type.Unit),
+                                    Val.Global(name, Type.Ref(name)),
+                                    Seq()),
+                            Next.None)
```

Comment:
The previous code used an `unwind` handler here. I'm pretty sure we still need one. What if one of the static initializers throws an exception?